tsfile/src/main/java/org/apache/iotdb/tsfile/write/chunk/ChunkWriterImpl.java

```diff
@@ -139,21 +139,21 @@ private void checkSdtEncoding() {
 
       if (isSdtEncoding && measurementSchema.getProps().containsKey("compmin")) {
```

Comment:
Don't forget to modify the most important code here. Please have a manual test.

`compmin` -> `compmintime`